actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -88,7 +88,14 @@ def translate(key, options = {})
           raise e if raise_error
 
           keys = I18n.normalize_keys(e.locale, e.key, e.options[:scope])
-          content_tag('span', keys.last.to_s.titleize, :class => 'translation_missing', :title => "translation missing: #{keys.join('.')}")
+          title = "translation missing: #{keys.join('.')}"
+
+          interpolations = options.except(:default)
```

Comment:
Fixed in https://github.com/rails/rails/commit/0fb2d1a0ba3360e928a7ac9d20378d7f5fbf915f

Thanks for the info, @arthurnn :grin: